SoftLayer/CLI/order/place.py

```diff
@@ -23,6 +23,9 @@
 @click.option('--verify',
               is_flag=True,
               help="Flag denoting whether or not to only verify the order, not place it")
+@click.option('--quantity',
+              type=int,
+              help="The quantity of the item being ordered")
```

Comment:
add a default of 1 here, so users just ordering 1 item don't have to specify that option

Reply:
Done